packages/test/test-end-to-end-tests/src/test/mapEndToEndTests.spec.ts

```diff
@@ -25,6 +25,16 @@ const testContainerConfig: ITestContainerConfig = {
     registry,
 };
 
+const getDefaultTestObject = async (container: Container): Promise<ITestFluidObject>=>{
```

Comment:
example of back compat host code in a test. it works in most cases, but didn't make it complete